procedures/auxtel/observation_procedures/DayTime-Checkout.ipynb

```diff
@@ -5,10 +5,10 @@
    "id": "3dbf3c0c",
```

Comment:
<p>This isn't using asyncio so it can just be a normal function (remove the async and all the awaits when it's called).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/92/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>Yes! Thank you for catching this up.</p>